docs/hardware/antenna/antenna-report.mdx

```diff
@@ -0,0 +1,13 @@
+---
+id: antenna-reports
+title: testing done on common antennas
+sidebar_label: antenna-report
+slug: /hardware/antenna-report
+sidebar_position: 2
+---
+
+
+
+### Meshtastic-Antenna-Reports
+
+RichInNewMexico has gone throught the trouble of testing a number of commonly purchaed antennas in the meshtastic community and given an opinion on weather or not a given antenna is worth using. please check out the project at [Meshtastic-Antenna-Reports](https://github.com/RicInNewMexico/Meshtastic-Antenna-Reports)
```

Comment:
Looks like a good start. Could we fix the following typos?

`RicInNewMexico` has gone `through` [...] `purchased` antennas in the `Meshtastic` community [...] `whether`

I'd like to phrase the latter half of the sentence in the positive. `whether or not a given antenna is worth using.` Might go something like `whether or not an antenna has performed optimally.`